streams/src/test/java/org/apache/kafka/streams/processor/internals/PartitionGroupTest.java

```diff
@@ -140,49 +140,47 @@ public void testTimeTracking() {
         record = group.nextRecord(info);
         // 1:[5, 2, 4]
         // 2:[4, 6]
-        // st: 3 (2's presence prevents it from advancing to 4)
```

Comment:
Why do we remove the comment? Should we rather just update it?

Reply:
ack, will add some comment